src/Json.Schema.ToDotNet/DataModelGenerator.cs

```diff
@@ -72,7 +72,7 @@ public string Generate(JsonSchema rootSchema)
                 throw Error.CreateException(Resources.ErrorNotAnObject, rootSchemaType.ToString().ToLowerInvariant());
             }
 
-            string rootFileText = GenerateClass(_settings.RootClassName, _rootSchema, _settings.SealClasses, _settings.TypeNameSuffix);
```

Comment:
> _settings.SealClasses, _settings.TypeNameSuffix) [](start = 85, length = 49)

No need to pass these properties in because this method is _always_ called (there are only two call sites) with these arguments. So the called method can access the instance field `_settings` directly. I noticed that when I was about to add `_settings.VirtualProperties` to the already long argument list.